File: app/src/main/java/com/mrk/mahfuj/bangladeshsonbidhan/JsonToJava/Law.java

package com.mrk.mahfuj.bangladeshsonbidhan.JsonToJava;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Law {

    @SerializedName("Title")
    @Expose
    private String title;
    @SerializedName("Content")
    @Expose
    private List<Content> content = null;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public List<Content> getContent() {
        return content;
    }

    public void setContent(List<Content> content) {
        this.content = content;
    }

}

File: app/src/main/java/com/mrk/mahfuj/bangladeshsonbidhan/Law.kt
package com.mrk.mahfuj.bangladeshsonbidhan

import java.util.ArrayList

open class Law {
    open var Title: String = ""
    open var Containts: ArrayList<Content>? = null
    open var ContentTypeTitle :String =""

    constructor(Title: String) {
        this.Title = Title
    }

    constructor(Title: String, ContentTypeTitle : String, Constants: ArrayList<Content>?) {
        this.Title = Title
        this.Containts = Constants
        this.ContentTypeTitle = ContentTypeTitle
    }

}